manager/integration/tests/test_cloning.py

```diff
@@ -392,7 +393,11 @@ def test_cloning_interrupted(client, core_api, pvc, pod, clone_pvc, clone_pod):
     crash_replica_processes(client, core_api, source_volume_name)
 
     # Step-7
-    wait_for_volume_faulted(client, source_volume_name)
+    # This is a workaround, since in some case it's hard to
+    # catch faulted volume status
+    wait_for_volume_status(client, source_volume_name,
```

Comment:
Does the volume starts to attach immediately after becoming faulted?

Also a qq, why are we waiting for volume to become degraded before we are crashing all the replicas in step 6?

Reply:
Hi @khushboo-rancher ,

Yes, in master-head volume will start to attach after faulted.
In step6, I think it detect new created clone volume, clone volume's replica are not all ready so it will be in degraded status